Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextBoxRegexEx/TextBoxRegexEx.bind

```diff
@@ -0,0 +1,94 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.TextBoxRegexExPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:common="using:Microsoft.Toolkit.Uwp.SampleApp.Common"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+  <Page.Resources>
+    <common:BoolStringConverter x:Key="StringFormatConverter" />
+    <Style x:Key="TextBoxRegexExStyle"
+           TargetType="TextBox">
+      <Setter Property="VerticalAlignment" Value="Top" />
+      <Setter Property="TextWrapping" Value="Wrap" />
+    </Style>
+    <DataTemplate x:Key="HeaderTemplate">
+      <StackPanel>
+        <TextBlock Text="{Binding}"
+                   TextWrapping="WrapWholeWords" />
+      </StackPanel>
+    </DataTemplate>
+  </Page.Resources>
+
+  <Grid Background="{StaticResource Brush-Grey-05}">
+    <Grid Margin="30">
+      <Grid.RowDefinitions>
+        <RowDefinition />
+        <RowDefinition />
+        <RowDefinition />
+        <RowDefinition />
+      </Grid.RowDefinitions>
+
+      <StackPanel Margin="10,0,10,0">
+        <TextBox Name="PhoneNumberValidator"
+                 controls:TextBoxRegexEx.Regex="^\s*\+?\s*([0-9][\s-]*){9,}$"
+                 Header="Text box with Regex extension for phone number, validation occurs on textbox changed"
+                 HeaderTemplate="{StaticResource HeaderTemplate}"
+                 Style="{StaticResource TextBoxRegexExStyle}"
+                 Text="" />
+        <StackPanel Orientation="Horizontal">
+          <TextBlock Text="Validation Result: " />
```

Comment:
Change the text to "Is Valid", or "Valid Value" or similar. With "Validation Result" I would expect values of "Valid" and "Invalid"